app/src/main/java/crux/bphc/cms/fragments/MyCoursesFragment.java

```diff
@@ -285,34 +285,36 @@ public void onFailure(String message,Throwable t) {
     }
 
     private void updateCourseContent (List<Course> courses) {
-        List<Course> newCourses = courseDataHandler.setCourseList(courses);
+        courseDataHandler.setCourseList(courses);
         CourseRequestHandler courseRequestHandler = new CourseRequestHandler(getActivity());
+        final int[] coursesUpdated = new int[1];
```

Comment:
Instead of doing this, just make it a global `int` variable. Should work, and is more elegant.